evm_arithmetization/src/cpu/kernel/interpreter.rs

```diff
@@ -382,117 +341,145 @@ impl<'a, F: Field> Interpreter<'a, F> {
         self.set_memory_multi_addresses(&block_hashes_fields);
     }
 
-    fn checkpoint(&self) -> InterpreterCheckpoint {
-        let registers = InterpreterRegistersState {
-            kernel_mode: self.is_kernel(),
-            context: self.context(),
-            registers: self.generation_state.registers,
+    fn interpreter_pop<const N: usize>(&mut self) -> Result<[U256; N], ProgramError> {
```

Comment:
What's the difference with `pop`? Maybe add some comment?

Reply:
Actually, it's not used anymore.. I missed those interpreter functions when cleaning up, sorry!